scripts/setup_gemdaq.sh

```diff
@@ -43,17 +58,20 @@ do
             VENV_ROOT="$OPTARG";;
         w)
             WIPE=1;;
+        P)
```

Comment:
add this to `helpstring` with an appropriate description.

Reply:
It is there, isn't it? l#25

Reply:
It was added to the helpstring as requested.